test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/AddressRegistrationTests.cs

```diff
@@ -149,15 +149,13 @@ public class AddressRegistrationTests
                 host.Start();
 
                 Assert.Equal(5000, host.GetPort());
-
-                var debugLog = testLogger.Messages.Single(log => log.LogLevel == LogLevel.Debug &&
-                    !log.Message.StartsWith("Hosting start", StringComparison.Ordinal)); // Filter out "Hosting starting" and "Hosting started"
-                Assert.True(string.Equals(debugLog.Message, $"No listening endpoints were configured. Binding to {Constants.DefaultIPEndPoint} by default.", StringComparison.Ordinal));
+                Assert.Equal($"No listening endpoints were configured. Binding to {Constants.DefaultIPEndPoint} by default.",
```

Comment:
That depends on the original intention of the test. I read it as, there is only one message from Kestrel and it's the binding to default address test. With this change, it becomes, there's is only one binding to default address message but I don't care if there are other messages from Kestrel. Functionally, there's no difference since right now there is only one message on startup.